src/main/resources/tags/errors/logintoxbox.tag

```diff
@@ -0,0 +1,12 @@
+type: text
+issues: Please log into Xbox to join this server.
+
+---
+
+This error occurs when Geyser is unable to validate your login. This might be because the client didn't send the right data, or you just aren't logged in.
+You can use the following steps to debug this:
```

Comment:
```suggestion
This occurs when Geyser cannot validate your login. Follow these steps to debug the issue:
```

Shorter is usually better, especially with the type of audience that's scared of reading /s